30_Day_Leetcode_Challenge/Week 3/Day15_ProductOfArrayExceptSelf.py

```diff
@@ -0,0 +1,25 @@
+class Solution:
+    def productExceptSelf(self, nums: List[int]) -> List[int]:
```

Comment:
I love that you alr start to use type hinting in the code, its a very good practice.
However, you will need to import List from typing package.
from typing import List